Wireframe/css/style.css

```diff
@@ -0,0 +1,119 @@
+*{
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  height: 100vh;
+}
+
+main {
+  width: 80%;
+  margin: 1rem auto;
+  height: 100vh;
+}
+section {
+  width: 100%;
+  margin: 0 auto;
+}
+header {
+  padding: 1rem 0;
+  border-bottom: 1px solid rgba(0, 0, 0, 0.5)
+}
+
+nav {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  padding: 1rem;
+}
+
+nav h2 {
+  color: #6655bf;
+}
+
+nav #nav-link{
+  display: flex;
+  justify-content: space-around;
+  gap: 1.5rem;
+  font-size: 1.3rem;
+}
+
+#nav-link a {
+  text-decoration: none;
+  color: #6655bf;
+}
+
+details#git-definition:first-child {
```

Comment:
Since the value of an `id` attribute is unique, it is redundant to to specify also the element name.
`details#git-definition` is the same as just `#git-definition`.

Anyway, this look like a "forced" use of `:first-child`. Maybe you can try the other pseudo-class selector that can select the first `<article>` element in a page (or first element of a particular type among all elements that share the same ancestor)